compiler/ghc/default.nix

```diff
@@ -118,6 +118,10 @@ let
   '' + lib.optionalString enableRelocatedStaticLibs ''
     GhcLibHcOpts += -fPIC
     GhcRtsHcOpts += -fPIC
+    GhcRtsCcOpts += -fPIC
+  '' + lib.optionalString (enableRelocatedStaticLibs && targetPlatform.isx86_64) ''
+     GhcLibHcOpts += -fexternal-dynamic-refs
+     GhcLibHcOpts += -fexternal-dynamic-refs
```

Comment:
This is a typo, surely?

I've had success building a statically linked `x86_64-unknown-linux-musl` binary using `projectCross.musl64` and `ghc923` with [this version](cad8ee71120153efb0652fafe9024261f977504f).